dracut/anaconda-ifdown

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# Turn off given network interface and remove all its flags from Dracut.
+#
+# Author: Jiri Konecny
+#
+
+netif="$1"
+# ip down/flush ensures that routing info goes away as well
+ip link set $netif down
+ip addr flush dev $netif
+echo "#empty" > /etc/resolv.conf
+rm -f -- /tmp/*.$netif.*
+if [ -e /sys/class/net/$netif/address ]; then
+    address=$(cat /sys/class/net/$netif/address)
+    rm -f -- /tmp/*.$address.*
+fi
```

Comment:
The ``rm`` utility fails if there's nothing to remove, doesn't it? Not sure if that's desirable here and above.